src/scopes.py

```diff
@@ -20,6 +21,7 @@ def outer():
 
     def inner():
         y = 999
+        print(y)
```

Comment:
Similar to above, you can change y without having to move the print statement with the `nonlocal` keyword.